src/sentry/static/sentry/app/views/organizationGroupDetails/header.jsx

```diff
@@ -103,32 +102,37 @@ class GroupHeader extends React.Component {
                 <EventOrGroupTitle data={group} />
               </GuideAnchor>
             </h3>
-            <div className="event-message">
-              <span className="error-level">{group.level}</span>
-              {message && <span className="message">{message}</span>}
-              {group.logger && (
-                <span className="event-annotation">
-                  <Link
-                    to={{
-                      pathname: baseUrl,
-                      query: {query: 'logger:' + group.logger},
-                    }}
-                  >
-                    {group.logger}
-                  </Link>
-                </span>
-              )}
-              {group.annotations.map((annotation, i) => {
-                return (
-                  <span
-                    className="event-annotation"
-                    key={i}
-                    dangerouslySetInnerHTML={{__html: annotation}}
-                  />
-                );
-              })}
-            </div>
+
+            <EventMessage
+              message={message}
+              level={group.level}
+              annotations={
+                <React.Fragment>
+                  {group.logger && (
+                    <EventAnnotation>
+                      <Link
+                        to={{
+                          pathname: baseUrl,
+                          query: {query: 'logger:' + group.logger},
+                        }}
+                      >
+                        {group.logger}
+                      </Link>
+                    </EventAnnotation>
+                  )}
+                  {group.annotations.map((annotation, i) => {
+                    return (
```

Comment:
tiny nit, can save one level of indentation here with implicit return.

Reply:
can we add an eslint rule for this

Reply:
https://eslint.org/docs/rules/arrow-body-style yup. WIll add